src/nix-build/nix-build.cc

```diff
@@ -305,8 +305,9 @@ static void main_nix_build(int argc, char * * argv)
                                         inShebang && !packages ? absPath(i, absPath(dirOf(script))) : i)))));
             }
         }
-
+    logger->stopActivity(parseActivity.id);
     /* Evaluate them into derivations. */
+    auto evaluateActivity = Activity(*logger, lvlTalkative, actUnknown, "evaluate expressions into derivations");
```

Comment:
```suggestion
    Activity evaluateActivity(*logger, lvlTalkative, actUnknown, "evaluate expressions into derivations");
```
and maybe this should say "store derivations"?